### DISCUSSION TITLE
Iframe embed has no padding?

### DISCUSSION THREAD

**Discussion by BernhardBaumrock** [plausible/analytics#2764] [Anything Else]
I have developed a module for the ProcessWire CMS to integrate the plausible dashboard into the CMS backend. Unfortunately it seems that the embed code has changed somehow so that my iframe does not have any padding on its sides:

<img width="1272" alt="image" src="https://user-images.githubusercontent.com/8488586/225736555-e15726e9-98c9-4f03-ac42-e4b407595038.png">

I've read the docs, but &width=manual does also not work. I only get some padding if I remove the &embed=true, but then I get the plausible logo and login links, which I don't want to see (as they don't work in the iframe!).

Thx in advance!

**Comment by ukutaht** [comment]
Yes we recently made a change to embedded dashboards: https://github.com/plausible/analytics/pull/2744/files

You should be able to get the old behaviour by using the manual width mode and then creating a container for the embed:

```html
<div style="max-width: 1024px; margin-left: auto; margin-right: auto;">
  <!--embed code with width=manual-->
</div>
```

**Comment by BernhardBaumrock** [comment]
Hi @ukutaht  thx for your response! Your suggestion does not work.

Here is how your code looks like:
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/8488586/226868979-e9cf4d55-8e29-4c28-817b-4e28efa8d376.png">

You see that there is no padding on the right and left WITHIN the iframe!

This is the code used to embed the iframe:

```php
<div style="max-width: 1024px; margin-left: auto; margin-right: auto;">
        <iframe plausible-embed src="' . $url . '&width=manual" scrolling="no" frameborder="0"
        loading="lazy" style="width: 1px; min-width: 100%; height: 1600px;"></iframe>
      </div>
      <script async src="https://' . $host . '/js/embed.host.js"></script>
```

When adding &embed=false it looks like this and has proper padding:
<img width="1227" alt="image" src="https://user-images.githubusercontent.com/8488586/226869705-3449580d-da10-412d-9c54-3b359868e26a.png">

But as stated above the plausible logo and login link appearing on top of the iframe do not work (and I don't want them to show up).

Thx for your help!

PS: Here is a screenshot showing that even with background=transparent we would need some padding otherwise it looks cropped:
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/8488586/226871060-2825252d-2638-4e83-9b3c-25e853602a25.png">

**Comment by BernhardBaumrock** [comment]
@ukutaht do you need any more information from my side? Is the issue clear? Thx in advance.

**Comment by IttzyTT** [comment]
is it possible to change the color of the bar?

**Comment by BernhardBaumrock** [comment]
I'm closing this as I've happily switched to Matomo :)
